transform/combined_filter.go

```diff
@@ -0,0 +1,274 @@
+package transform
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/RoaringBitmap/roaring/roaring64"
+	"github.com/streamingfast/bstream"
+	"github.com/streamingfast/bstream/transform"
+	"github.com/streamingfast/dstore"
+	"github.com/streamingfast/eth-go"
+	pbtransform "github.com/streamingfast/sf-ethereum/types/pb/sf/ethereum/transform/v1"
+	pbeth "github.com/streamingfast/sf-ethereum/types/pb/sf/ethereum/type/v1"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/anypb"
+)
+
+const NP = ""  //
+const LP = "L" // log prefix for combined index
```

Comment:
Don't think it's worth it to have those so short, I don't see any problem having the long form.